### PR TITLE
fix: Glossary call format

### PR DESCRIPTION
**Pull request by nschonni**
- Remove duplicate name calls as separate paramater
- Removed 3rd parameter calls that Glossary doesn't have in Yari

### CODE REVIEW

File: files/en-us/glossary/ajax/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ tags:
   - l10n:priority
 ---
 
-**Ajax**, which initially stood for Asynchronous {{Glossary("JavaScript")}} And {{Glossary("XML")}}, is a programming practice of building complex, dynamic webpages using a technology known as {{Glossary("XHR_(XMLHttpRequest)","XMLHttpRequest")}}.
```
Comment (nschonni) [inline files/en-us/glossary/ajax/index.md:12]: Realized this rename is still pending in another PR

**Review by bsmth — APPROVED** [review]
Changes look good to me, thanks 👍🏻

**Comment by wbamberg** [comment]
@nschonni , is there a reason this is draft, or can we merge it?

**Review by estelle — APPROVED** [review]

**Comment by estelle** [comment]
@nschonni I know you were waiting for another PR to be merged. Just wanted to bump this as it has been a month.... not sure if the blocker is resolved.

**Comment by nschonni** [comment]
Sorry, I updated my comment above to link to the actual other stalled PR. This one has one rename for the `XHR_(XMLHttpRequest)`. I can rebase that bit out if you want to get this landed

**Comment by bsmth** [comment]
> For the `XHR_(XMLHttpRequest)`, I can rebase that bit out if you want to get this landed


Let's do this @nschonni, there are clear improvements here that would be great to get in!

**Comment by nschonni** [comment]
Realized that since there is an old redirect for `XMLHttpRequest` this will still work, the flaw is just complaining about the redirect, rather than fully failing.
Marked as ready

**Comment by bsmth** [comment]
Thanks @nschonni, merging 👍🏻

**Pull request merged**